pkg/cmd/release/create/create.go

```diff
@@ -78,13 +78,19 @@ func NewCmdCreate(f *cmdutil.Factory, runF func(*CreateOptions) error) *cobra.Co
 			git, push the tag to GitHub, then run this command.
 		`, "`"),
 		Example: heredoc.Doc(`
-			# use release notes from a file
+			Interactively create a release
```

Comment:
We used to prefix "comments" in examples with `#`. The changes here go the other way and remove `#` from non-code parts. Do we want to set this as a standard for all documentation?

Reply:
I prefer this format and find the # to be line noisy. I'd rather us standardize on this form but just realized i am not sure how it looks in the html manual...